internal/engine/upper.go

```diff
@@ -113,6 +112,7 @@ func (u Upper) Start(ctx context.Context, args []string, watchMounts bool) error
 	}
 
 	manifests, globalYAML, configFiles, err := loadAndGetManifests(ctx, manifestNames)
+	// ~~ maybe this is just for now...?
 	if err != nil {
```

Comment:
Yes we're not going to be error'ing out of here in the near future.

Reply:
cool -- am gonna leave it in this branch for now so that i can accurately debug things tho